app/src/main/java/ch/epfl/sweng/swenggolf/Offer.java

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.sweng.swenggolf;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Offer implements Parcelable {
```

Comment:
What are the motivation for the class to implements Parcelable?

Reply:
It's so it can be sent to another activity [using an Intent and .putExtra()](https://stackoverflow.com/questions/10107442/android-how-to-pass-parcelable-object-to-intent-and-use-getparcelable-method-of#10114932).

I use it to communicate between the list and the ShowOffer, but we can imagine using this feature for other activites of course.